src/main/java/frc/robot/commands/VarFeedPrime.java

```diff
@@ -0,0 +1,102 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import static edu.wpi.first.units.Units.*;
+
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.wpilibj.DriverStation;
+import edu.wpi.first.wpilibj.DriverStation.Alliance;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.constants.Constants;
+import frc.robot.subsystems.Elevator;
+import frc.robot.subsystems.Shooter;
+import frc.robot.subsystems.chassis.PoseEstimator;
+import java.util.function.DoubleSupplier;
+import org.littletonrobotics.junction.Logger;
+
+public class VarFeedPrime extends Command {
+  private Shooter shooter;
+  private Elevator elevator;
+  private PoseEstimator poseEstimator;
+  private Translation2d plopTranslation;
+  // private NetworkTableEntry lInput;
+
+  // Translation of where the note exits in the XZ plane (side view)
+  private final Translation2d shooterXZTrans;
+
+  private final DoubleSupplier wristAngle; // Range of velocity from min distance to max distance
+  private final double RPM_MPS;
```

Comment:
Save this as `MPS_TO_RPM` and set as `1 / RPM_MPS` that you get in the constructor